src/lib/components/compositions/radio/animated.svelte

```diff
@@ -27,35 +28,38 @@
   });
 </script>
 
-<div>
-  {#each options as option}
-    <div class="tw_input-spacing">
-      <input
-        aria-invalid={isInvalid}
-        bind:this={inputEl}
-        class="tw_radio-input_animated dark:tw_radio-input_animated_dark tw_sr-only"
-        checked={defaultOption === option.value}
-        id={`${key}-${option.value}`}
-        {name}
-        on:change={onChange}
-        required={isRequired}
-        type="radio"
-        value={option.value}
-      />
-      <Label
-        key={`${key}-${option.value}`}
-        classes="tw_input-spacing tw_grid tw_grid-cols-[2.5em_1fr] tw_relative"
-      >
-        <span class="tw_animated-radio dark:tw_animated-radio_dark" />
-        {option.text}
-      </Label>
-    </div>
-  {/each}
-  <!--
+<fieldset>
+  <legend>{groupMessage}</legend>
```

Comment:
Cool, I'll pull this in locally and run it to see how it looks.

Reply:
Pulled code locally, added radio as a variant of Choice in the `choice.stories.js` file, and ran Storybook. Looks like the group label needs just a touch of styling and support of dark mode.

Screenshot:
https://share.getcloudapp.com/eDu77jBG

Directly adding some tailwind classes will do the trick. These will be sufficient:

```
<legend class="tw_input-label dark:tw_input-label_dark tw_font-bold tw_mb-4">
```

I also observed that the render for the Composition story for radio is rendering `undefined`, so we should probably handle the falsey value by defaulting to empty string if the exported prop is not passed.

Screenshot:
https://share.getcloudapp.com/2Nu77wzO

This will do it:

```
export let groupLabel = '';
```

That way, when nothing is passed, it doesn't render `undefined`.

Reply:
Ah good spot. Thanks.